src/components/resolvedicon.jsx

```diff
@@ -33,13 +33,34 @@ export default function ResolvedIcon({ icon, width = 32, height = 32, alt = "log
   }
 
   // check mdi- or si- prefixed icons
-  const prefix = icon.split("-")[0]
-
+  const prefix = icon.split("-")[0];
+  const suffix = icon.split("-")[icon.split("-").length - 1];
+  
   if (prefix in iconSetURLs) {
-    // get icon source
+    // check whether icon ends with color code
+    if (!(suffix.match(`[#][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9]`) == null)) {
+
```

Comment:
```suggestion
    if (suffix.test(`[#][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9]`)) {

```

use `test` to check if a regex matches

Reply:
Done, and I also included upper case letters since the regex was case sensitive so far.